esp/src/src-react/components/Sasha.tsx

```diff
@@ -35,11 +35,31 @@ export const Sasha: React.FunctionComponent<SashaProps> = () => {
         // Implement getLastServerMessage function call
         break;
       case "restoreECLWorkUnit":
-        // Implement restoreECLWorkUnit function call
-        break;
+        sashaService.RestoreWU({
+          Wuid: wuid,
+          WUType: "ECL"
+        })
+          .then(response => {
+            console.log("restoreECLWorkUnit response", response.Result);
+            setResult(response.Result);
+          })
+          .catch(err => {
+            console.error(err);
```

Comment:
and replace these with the logger component, so it gets added to the UI's errors page. You could refer to those dali components you added a while back for what that looks like